ru/passwords.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    "password" => "Пароль должен быть не менее шести символов и совпадать с подтверждением.",
+    "password" => "Пароль должен содержать не менее шести символов и совпадать с полем подтверждения пароля.",
```

Comment:
Sorry, I don't understand what `bikeshedding` means. I have just tried to make translated phrases to be more pleasant to the ear. The same goes to the `after :date`, you mentioned further.